components/big_cartel/sources/new-order/new-order.mjs

```diff
@@ -0,0 +1,33 @@
+import app from "../../big_cartel.app.mjs";
+import common from "../common.mjs";
+
+export default {
+  key: "big_cartel-new-order",
+  name: "New Order Event",
+  description: "Emit new events when a new order is created. [See the docs here](https://developers.bigcartel.com/api/v1#get-all-orders)",
+  version: "0.0.1",
+  type: "source",
+  ...common,
+  props: {
+    app,
+    ...common.props,
+  },
```

Comment:
I think you can remove this and add the `app` prop in `sources/common.mjs` right?